apm/otel_monitor.go

```diff
@@ -191,11 +197,45 @@ func (m *monitor) getSpan(evt *event.CommandFinishedEvent) (trace.Span, bool) {
 	return span, ok
 }
 
+func (m *monitor) dbStatementAttributes(evt *event.CommandStartedEvent) ([]attribute.KeyValue, error) {
+	var attributes []attribute.KeyValue
+	command := m.cfg.CommandTransformerFunc(evt.Command)
+	if command == nil {
```

Comment:
i'd love if we could put a comment here too to explain why this wouldn't be an error (but i think this does make sense)

Reply:
done